src/Components/ImagesTable/Status.tsx

```diff
@@ -316,10 +324,22 @@ const ErrorStatus = ({ icon, text, error }: ErrorStatusPropTypes) => {
             <Panel isScrollable>
               <PanelMain maxHeight="25rem">
                 <div className="pf-u-mt-sm">
-                  <p>{reason}</p>
+                  <div className="pf-v5-c-code-block">
+                    <div className="pf-v5-c-code-block__header">{reason}</div>
+                    <div className="pf-v5-c-code-block__content">
+                      <pre className="pf-v5-c-code-block__pre">
+                        {detailsArray.join('\n')}
+                        <code className="pf-v5-c-code-block__code"></code>
```

Comment:
Not sure about using code blocks here, the result doesn't work well for more common errors:

DNF:
![image](https://github.com/osbuild/image-builder-frontend/assets/11140201/c4b9763f-7430-41d5-a091-56b2367880ae)

target errors:

![image](https://github.com/osbuild/image-builder-frontend/assets/11140201/5ef40975-7fa4-4def-bc90-e7202367a458)


We can maybe make the popover wider, and then have reason and details just separated by a vertical line or something?

If we can somehow tell when it's a stacktrace, putting that in a codeblock could be nice.

Reply:
totally agree - that's actually for someone else to decide - I'm far away from being a react or PatternFly expert!